packages/forms/frontend/sirius-components-forms/src/propertysections/PropertySectionLabel.tsx

```diff
@@ -38,13 +39,18 @@ const usePropertySectionLabelStyles = makeStyles((theme) => ({
     width: theme.spacing(2),
     height: theme.spacing(2),
   },
+  typography: {
+    lineHeight: '1.5',
```

Comment:
I have look at  https://v4.mui.com/components/typography/#typography for other style of typographie. I wanted to keep a "bold" style because it makes it clear this is the label of the widget so I kept "subtitle2". If you prefer I can use "body2" but I think I would make it bold. Do you think it is better?

Reply:
Ok, let's keep it that way.

Reply:
Ok